### PR TITLE
 main/dhcp: fix missing /var/run/dhcp for dhcrelay

### PR DESCRIPTION
**Pull request by liske**
This fixes issue [#10024](https://bugs.alpinelinux.org/issues/10024).

### CODE REVIEW

File: main/dhcp/dhcrelay.initd [hunk 1]
```diff
@@ -20,6 +20,8 @@ start() {
 		IFACES="${IFACES} -i ${i}"
 	done
 
+	checkpath -d -o dhcp:dhcp /var/run/dhcp
```
Comment (ncopa) [inline main/dhcp/dhcrelay.initd]: I prefer long options in scripts as they are more readable

Comment (liske) [reply main/dhcp/dhcrelay.initd]: @ncopa I've updated the PR to contain long options.

**Comment by liske** [comment]
Can we get this PR merged?

**Comment by tcely** [comment]
> Can we get this PR merged?

I would like to see #6265 merged instead. I agree this needs to be fixed, but merging this first will leave the other fixes out and cause more work for me.

**Pull request closed without merging**